src/frontend/src/pages/FinancePage/FinanceComponents/ReportRefundModal.tsx

```diff
@@ -41,15 +45,16 @@ const ReportRefundModal: React.FC<ReportRefundProps> = ({ modalShow, handleClose
   } = useForm({
     resolver: yupResolver(schema),
     defaultValues: {
-      refundAmount: ''
+      refundAmount: '',
+      dateReceived: new Date()
     },
     mode: 'onChange'
   });
 
-  const handleConfirm = async (data: { refundAmount: number }) => {
+  const handleConfirm = async (data: { refundAmount: number; dateReceived: Date }) => {
     handleClose();
     try {
-      await mutateAsync({ refundAmount: data.refundAmount * 100 });
+      await mutateAsync({ refundAmount: data.refundAmount * 100, dateReceived: data.dateReceived.toISOString() });
       toast.success(`New Account Credit Amount #${data.refundAmount} Reported Successfully`);
```

Comment:
Can you also just change this msg to be 'New Account Credit Reported Successfully'